config/routes.rb

```diff
@@ -12,6 +14,7 @@
   end
   resources :resource, :only => :show
   mount HydraEditor::Engine => '/'
+  mount Sidekiq::Web => '/sidekiq'
```

Comment:
Some security here would be good.

https://github.com/mperham/sidekiq/wiki/Monitoring#devise